app/vmctl/flags.go

```diff
@@ -728,6 +728,32 @@ var (
 	}
 )
 
+const (
+	backoffRetries     = "backoff-retries"
+	backoffFactor      = "backoff-factor"
+	backoffMinDuration = "backoff-min-duration"
```

Comment:
Agree with @zekker6: we should have 2 set of flags: `vm-backoff` and `vm-native-backoff` prefixed.

Reply:
We discussed with @zekker6 on meeting and decided to have 2 sets of flags, i will update the logic soon

Reply:
made a split of the global flags to the scopes updated tests